app/scripts/views/complete_sign_up.js

```diff
@@ -60,6 +64,17 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       return self.fxaClient.verifyCode(uid, code)
           .then(function () {
             self.logScreenEvent('verification.success');
+            var account = self.getAccount();
+
+            if (account.get('needsOptedInToMarketingEmail')) {
```

Comment:
An OAuth token needs to be generated for opt-in, in a 2nd browser the user will not have a sessionToken that can be used to generate an OAuth token. We can get around this in a couple of ways:

1) Have the auth server fetch an oauth token and opt in on behalf of the user.
2) Have the user authenticate in the 2nd browser, an oauth token can then be created, and the user opts in.

There are probably other ways, but of those two, I'm partial to the 2nd.